<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableSelectorDialog</class>
 <widget class="QDialog" name="VariableSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Variable Selection...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="selectbyregexpButton">
        <property name="text">
         <string>Select by match variable string name</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="regexp"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="selectbyvarlabelButton">
        <property name="text">
         <string> Select by variable labels</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="varlabelslst">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="selectbyObj">
        <property name="text">
         <string>Select by Object </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>234</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="2" column="2">
       <widget class="QLabel" name="selectedlabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QPushButton" name="findButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>128</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QListView" name="varnamelistView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="searchline"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="objectGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1" colspan="2">
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="min"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="max"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="valuerange">
            <property name="text">
             <string>Values in range</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_16">
          <item row="0" column="0">
           <widget class="QRadioButton" name="valueinlimits">
            <property name="text">
             <string>Value over limits</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>inferior (&lt;)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="limitinf"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>superior (&gt;)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="limitsup"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="variablelst">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QListView" name="objectlistView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>43</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="objlayerBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="selectButton">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="invertselectButton">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Invert Selection</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="appendselectionButton">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Append Selection...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="unselectButton">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Unselect</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>86</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>objectGroupBox</zorder>
  <zorder>selectButton</zorder>
  <zorder>appendselectionButton</zorder>
  <zorder>invertselectButton</zorder>
  <zorder>unselectButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
